### ISSUE TITLE
Add filter to Shows table to sort/show shows per missing episodes amount

### ISSUE THREAD

**Issue by BankaiNoJutsu** [mregni/EmbyStat#1646]
Hey!

Would it be possible to add a filter to Shows table in order to sort/show Shows per missing episodes amount?

Cheers!

**Comment by mregni** [comment]
I already added some filters and sorting, but didn't add sorting/filter on missing episodes yet. I'll see if I can add this one in the next release.

**Comment by BankaiNoJutsu** [comment]
Sure! Would be a great addition in order to quickly see which series have the most missing episodes. Do you also think this would be useful?

Thanks!

**Comment by mregni** [comment]
Yeah, would be very useful indeed. I just need to create some extra query parameters for it. With the new database structure, this might be fast to implement.
